app-connector-client/server/server.js

```diff
@@ -40,15 +41,27 @@ app.post("/connection", function (req, res) {
 
 
 });
+app.get("/connectioninfo", returnConnectionInfo)
+app.post("/connectioninfo", returnConnectionInfo)
```

Comment:
I think what #63 meant is returning connection information as a response to `connection` post request above.

Reply:
No, because you might have a connection to Kyma cluster. Than you stop and restart the server and your connection persists. In this case `/connection` is never called but you still need info of your current connection.